backend/LeetWars.Notifier/LeetWars.Notifier.WebAPI/Models/CodeRunResults.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace LeetWars.Notifier.WebAPI.Models
+{
+    public class CodeRunResults
+    {
+        public long UserId { get; set; }
+        public long ChallengeVersionId { get; set; }
+        public string Language { get; set; } = string.Empty;
+        public bool IsBuilt { get; set; } = false;
```

Comment:
the default value of a bool is false. i think u can remove it

Reply:
Removed as well